unified-codes/package.json

```diff
@@ -31,20 +31,20 @@
   },
   "private": true,
   "dependencies": {
-    "@apollo/client": "^3.2.1",
-    "@babel/plugin-transform-runtime": "^7.11.5",
+    "@apollo/client": "^3.2.4",
```

Comment:
Wondering if we should look at removing this one now that we aren't using it?

Reply:
Good idea. I wonder if there is a tool for trimming unused deps?

Reply:
Update: [woop woop!](https://github.com/depcheck/depcheck) (note: seems to not work v. well for `devDependencies`, so be careful).

Reply:
6aa82b1c4521dac3eeede8c3f1225a496c367ffa. Also looks like some deps were missing from the `package.json`, possibly automatically installed by other packages... but acb6918013b1bcfa7a878552665688736b618190 just to be safe.

![safe](https://user-images.githubusercontent.com/43223637/97153994-3f75ed80-17d8-11eb-8b10-78c2123f0ae9.gif)